types/node/index.d.ts

```diff
@@ -1296,24 +1285,24 @@ declare module "http" {
         destroy(): void;
     }
 
-    export var METHODS: string[];
+    const METHODS: string[];
 
-    export var STATUS_CODES: {
+    const STATUS_CODES: {
         [errorCode: number]: string | undefined;
         [errorCode: string]: string | undefined;
     };
 
-    export function createServer(requestListener?: (request: IncomingMessage, response: ServerResponse) => void): Server;
-    export function createClient(port?: number, host?: string): any;
+    function createServer(requestListener?: (request: IncomingMessage, response: ServerResponse) => void): Server;
+    function createClient(port?: number, host?: string): any;
 
     // although RequestOptions are passed as ClientRequestArgs to ClientRequest directly,
     // create interface RequestOptions would make the naming more clear to developers
-    export interface RequestOptions extends ClientRequestArgs { }
-    export function request(options: RequestOptions | string | URL, callback?: (res: IncomingMessage) => void): ClientRequest;
-    export function request(url: string | URL, options: RequestOptions, callback?: (res: IncomingMessage) => void): ClientRequest;
-    export function get(options: RequestOptions | string | URL, callback?: (res: IncomingMessage) => void): ClientRequest;
-    export function get(url: string | URL, options: RequestOptions, callback?: (res: IncomingMessage) => void): ClientRequest;
-    export var globalAgent: Agent;
+    interface RequestOptions extends ClientRequestArgs { }
+    function request(options: RequestOptions | string | URL, callback?: (res: IncomingMessage) => void): ClientRequest;
+    function request(url: string | URL, options: RequestOptions, callback?: (res: IncomingMessage) => void): ClientRequest;
+    function get(options: RequestOptions | string | URL, callback?: (res: IncomingMessage) => void): ClientRequest;
+    function get(url: string | URL, options: RequestOptions, callback?: (res: IncomingMessage) => void): ClientRequest;
+    const globalAgent: Agent;
```

Comment:
`globalAgent` is not `const`.

Reply:
Seems that way, however changing it on the fly could break references.
I will change it to let.